Apps/W1/HybridGP/app/src/Migration/Support/HelperFunctions.codeunit.al

```diff
@@ -1159,13 +1159,19 @@ Codeunit 4037 "Helper Functions"
 
     procedure GetNumberOfItems(): Integer;
     var
-        GPItem: Record "GP Item";
         GPCompanyAdditionalSettings: Record "GP Company Additional Settings";
+        GPIV00101: Record "GP IV00101";
     begin
         if not GPCompanyAdditionalSettings.GetInventoryModuleEnabled() then
             exit(0);
 
-        exit(GPItem.Count());
+        if not GPCompanyAdditionalSettings.GetMigrateInactiveItems() then
+            GPIV00101.SetRange(INACTIVE, false);
+
+        if not GPCompanyAdditionalSettings.GetMigrateDiscontinuedItems() then
+            GPIV00101.SetFilter(ITEMTYPE, '<>%1', 2);
```

Comment:
This one is hardcoded. Could we use the other function or a method?

Reply:
Good point. I've committed a change, let me know if you think it could be done better.